### DISCUSSION TITLE
Welcome to Medusa Discussions!

### DISCUSSION THREAD

**Discussion by HanSolo** [HanSolo/Medusa#208] [Announcements]
Aloha everyone, in case of questions or ideas, feel free to discuss these topics here... :)

**Comment by ChasDouglass** [comment]
How can I change the color of a gauge on the fly?  I'm using the LCD skin and I'd like to change the gauge color to yellow or red to flag different conditions.  I tried changing various color settings (like foregroundBaseColor and valueColor) but they don't seem to update, the default color doesn't change.

**Reply by HanSolo** [reply]
Could you please provide a code snippet that shows the described behavior? That would make things much easier to reproduce for me.

**Reply by ChasDouglass** [reply]
That helped me work through it (as explaining a problem to someone else so often does).
I was trying to change the background color and value color, I've discovered what I really needed to do was change the LcdDesign colors.  That works!

`

**Comment by mamadece** [comment]
I want to try out and use the medusa library in Android Studio in Java. Just in a new project from scratch. I am completely newbie in Android.
I even cannot get the library into my project. I tried:
settings.gradle:
`   repositories {
        gradlePluginPortal()
        google()
        mavenCentral()
        maven { url 'https://jitpack.io' }
    }
`

build.gradle (app);
`dependencies {
    implementation "eu.hansolo.medusa:medusa:17.1.5"
}`
which I created based on how similar calls look like. I never saw the real one...

The following error is produced:
` > Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not resolve eu.hansolo.medusa:medusa:17.1.5.
     Required by:
         project :app
      > Could not resolve eu.hansolo.medusa:medusa:17.1.5.
         > Could not get resource 'https://jitpack.io/eu/hansolo/medusa/medusa/17.1.5/medusa-17.1.5.pom'.
            > Could not GET 'https://jitpack.io/eu/hansolo/medusa/medusa/17.1.5/medusa-17.1.5.pom'.
               > Read timed out`

Which repository and implementation do I have to use?

Dieter
P.S.: As this is my first question here: I have no idea why the original code formatting is discarded (newlines removed).
And - I will have more questions later, hope I'm welcome to issue them....

**Reply by HanSolo** [reply]
Well the reason why it doesn‘t work is pretty simple…it‘s a JavaFX library and Android does not support JavaFX. So you cannot use it on Android.

**Reply by mamadece** [reply]
What a pity! Thanks!
Any suggestions as alternative?

**Reply by DJViking** [reply]
You could look into the GluonFX Plugin to get JavaFX to work on Android. It uses GraalVM.
https://github.com/gluonhq/gluonfx-gradle-plugin

**Comment by abraham-ny** [comment]
Is there documentation for this library?

**Comment by HanSolo** [comment]
Nope, but you can learn from the demo project https://github.com/HanSolo/medusademo
